docs/models.rst

```diff
@@ -79,3 +79,19 @@ It is also possible to have two concrete Heroku Connect models to Inherit from
 each other.
 
 .. _`Multi-table inheritance`: https://docs.djangoproject.com/en/stable/topics/db/models/#multi-table-inheritance
+
+
+Signals
+-------
+
+Standard `Django model signals`_ won't be triggered in case of changes coming from Salesforce. This is happening because
+Heroku Connect operates on a database level.
+If you want to hook up to changes coming from Salesforce, there are several possible solutions:
+
+* Use `Salesforce PushTopic Events`_
+* Set up `Salesforce Outbound message`_ (triggered on object change) and a Django SOAP endpoint listening to it. From there You could make a Django signal to hook up.
```

Comment:
```suggestion
* Set up a `Salesforce Outbound message`_ (triggered on object change) and a Django SOAP endpoint listening to it. From there, a Django signal can be triggered.
```

Maybe add a note, that this would be asynchronous to the Heroku Connect db change and may cause race conditions.

Reply:
```suggestion
* Set up `Salesforce Outbound message`_ (triggered on object change) and a Django SOAP endpoint listening to it. From there you could make a Django signal to hook up.
```